test/plugin/test_out_google_cloud.rb

```diff
@@ -191,6 +191,46 @@ def test_parse_severity
     assert_equal('DEFAULT', test_obj.parse_severity('er'))
   end
 
+  # For non-grpc path, log name is reflected in the url, so we are stubbing the
+  # corresponding url containing the converted tag. To verify the tag is
+  # converted properly, we simply check the log entry is recorded successfully,
+  # and no exception (e.g. stubbing missing for certain url) is thrown.
+  def test_tag_acceptance
```

Comment:
There is a lot of duplication between this and the same test in `test_out_google_cloud_grpc.rb`. Can we factor some of it out into a common method?